rotkehlchen/api/v1/resources.py

```diff
@@ -2797,3 +2797,10 @@ class EventDetailsResource(BaseMethodView):
     @use_kwargs(get_schema, location='json_and_query')
     def get(self, identifier: int) -> Response:
         return self.rest_api.get_event_details(identifier=identifier)
+
+
+class AllEvmChainsResource(BaseMethodView):
+
+    @require_loggedin_user()
```

Comment:
nitpick: perhaps this endpoint should not be restricted for logged in users only?

Reply:
yeah, sure. Done